src/test/java/org/opentripplanner/routing/street/CarRoutingTest.java

```diff
@@ -130,12 +179,14 @@ private static String computePolyline(Graph graph, GenericLocation from, Generic
     var temporaryVertices = new TemporaryVerticesContainer(graph, request);
     final RoutingContext routingContext = new RoutingContext(request, graph, temporaryVertices);
 
-    var gpf = new GraphPathFinder(new Router(graph, RouterConfig.DEFAULT, Metrics.globalRegistry));
+    var gpf = new GraphPathFinder(
+      new Router(graph, transitModel, RouterConfig.DEFAULT, Metrics.globalRegistry)
+    );
     var paths = gpf.graphPathFinderEntryPoint(routingContext);
 
     GraphPathToItineraryMapper graphPathToItineraryMapper = new GraphPathToItineraryMapper(
-      graph.getTimeZone(),
-      new AlertToLegMapper(graph.getTransitAlertService()),
+      transitModel.getTimeZone(),
```

Comment:
You could do the same as the in the bicycle test above.

Reply:
Test simplified